src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable */
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
   );
 }
```

Comment:
- [ ] As stated the project requirement you should Use class based components (All components should be class-based). So kindly make your App.js component class-based.

![Capture](https://user-images.githubusercontent.com/93598189/206214940-76c0b713-0830-4c80-92ec-1f037d6bb450.PNG)

Reply:
Done. Now it's class component.